src/Identity/EntityFrameworkCore/src/UserOnlyStore.cs

```diff
@@ -311,7 +312,8 @@ public async override Task<IList<Claim>> GetClaimsAsync(TUser user, Cancellation
                 throw new ArgumentNullException(nameof(user));
             }
 
-            return await UserClaims.Where(uc => uc.UserId.Equals(user.Id)).Select(c => c.ToClaim()).ToListAsync(cancellationToken);
+            // TODO: Fix once EF query works
```

Comment:
Include an issue link with every TODO

Reply:
Issues don't exist yet, and it's more urgent to get this in. I will make sure there are issues.